### DISCUSSION TITLE
unreal mode

### DISCUSSION THREAD

**Discussion by skull-squadron** [open-watcom/open-watcom-v2#987] [Q&A]
Are there any MCEs of switching to unreal mode, e.g., flat addressing without protections?

1. Switch to protected mode
2. Tweak the GDT/LDT
3. Load CS from a descriptor that has D = 1
4. Switch back to real mode

**Comment by joncampbell123** [comment]
There's a DOSLIB library for flat real mode: https://github.com/joncampbell123/doslib/tree/master/hw/flatreal
